packages/kotti-ui/source/kotti-button-group/types.ts

```diff
@@ -1,7 +1,7 @@
 import { SpecifyRequiredProps } from '../types/utilities'
 
 export namespace KottiButtonGroup {
-	export type PropsInternal = {}
+	export type PropsInternal = Record<string, never>
```

Comment:
I rebased on current master and this line was making trouble. Apparently `{}` is a deprecated way to declare an empty object acording to `eslint`. Maybe something that's new in TS 4?

Reply:
Look up the rule, it's because {} or object basically fit literally anything in JS. I'm not sure what the best way to declare an empty object is, but usually it isn't actually an empty object and instead is a record or an object with lots of optional values

Reply:
I think the `Record<string, never>` is the best way to type it BTW. Good find.